test/generator_yaml/test_schemata.sh

```diff
@@ -0,0 +1,39 @@
+schemadir="../../support/schemas"
+all_valid_yaml=(../../doc ../../skel/problem/ ../problems valid_yaml)
+
+any_failed=0
+
+# Iterate over each .yaml file in the directory
+for dir in "${all_valid_yaml[@]}"; do
+    for file in $(find "$dir" -type f -name '*generators.yaml'); do
+	    echo -n "Processing $file... "
+	    output=$(cue vet "$file" $schemadir/*.cue -d "#Generators" 2>&1)
+	    exit_code=$?
+	    if [ $exit_code -eq 0 ]; then
```

Comment:
Both here and below: you may want to check for 0, 1, and anything else, to catch cue crashes? (But that should be rare so maybe not worth the effort.)